src/main/java/net/revelc/code/impsort/ImpSort.java

```diff
@@ -127,7 +127,10 @@ public Result parseFile(final Path path, final byte[] buf) throws IOException {
     ParseResult<CompilationUnit> parseResult =
         new JavaParser(new ParserConfiguration().setLanguageLevel(languageLevel)).parse(file);
     CompilationUnit unit = parseResult.getResult()
-        .orElseThrow(() -> new ImpSortException(path, Reason.UNABLE_TO_PARSE));
+        .orElseThrow(() -> {
+          parseResult.getProblems().forEach(System.out::println);
```

Comment:
Printing directly to System.out is not a great idea. Using the Maven plugin log would be a better option. These problems could also be added to the ImpSortException in some way.

Reply:
Hi, thanks for your comments.
Will change System.out to be Maven plugin log and add it to ImpSortException.